modules/flocIdSystem.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * This module adds flocId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/flocId
+ * @requires module:modules/userId
+ */
+
+import * as utils from '../src/utils.js'
+import {submodule} from '../src/hook.js'
+
+const MODULE_NAME = 'flocId';
+
+/**
+ * Add meta tag to support enabling of floc origin trial
+ * @function
+ * @param {string} token - configured token for origin-trial
+ */
+function enableOriginTrial(token) {
+  const tokenElement = document.createElement('meta')
```

Comment:
maybe you want to end lines with `;` in this function as well, just to be consistent with the rest of the file.

Reply:
I meant all the statements in this function, not just on this line :)